src/App.js

```diff
@@ -1,16 +1,32 @@
-import React from 'react';
+import React , {useState} from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+
+
+
 
 const App = () => {
+  const [likes, setLikes] = useState(0);
+
+
+  const updateLikes = (clicked, setClicked) => {
+    setClicked(!clicked);
+    
+    !clicked ?setLikes((p) => p + 1):setLikes((p) => p - 1);
+  }
```

Comment:
👍  This is a clever approach for updating likes! For each chat entry, it is only the icon that changes. The 'liking' is really only cosmetic which is a good argument against NOT updating the `chatMessages` object using state. 

However, we would use a different approach if our chat data was coming from a database that stored whether a chat was liked or not. This would likely involve incorporating other requests like PUT/PATCH whenever a chat was liked.